graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -284,10 +290,12 @@ public LookupTableApi createTable(@ApiParam LookupTableApi lookupTable) {
     }
 
     @PUT
-    @Path("tables")
+    @Path("tables/{idOrName}")
```

Comment:
Is this strictly required? I tried to avoid putting the ID in there twice,  because then we should also check that the ID in the payload is the same as in  the URL.

Also `idOrName` in the permission.